docs/static/command-line-flags.asciidoc

```diff
@@ -46,16 +46,13 @@ added[5.0.0-alpha3, Command-line flags have dots instead of dashes in their name
 *`--log.level LEVEL`*::
  Valid levels are:
 * `warn`: log warnings
-* `quiet`: log errors
-* `verbose`: log verbose info (for users)
+* `error`: log errors
+* `info`: log verbose info (for users)
 * `debug`: log debugging info (for developers)
```

Comment:
can you add `fatal` and `trace` level here ?
can you inverse `warn` and `error` lines ?